modules/manage/pages/kubernetes/manage-topics.adoc

```diff
@@ -0,0 +1,489 @@
+      source: DEFAULT_CONFIG
+      unknownTags: {}
+      value: "1209600000"
+kind: List
+metadata:
+  resourceVersion: ""
+----
+====
+
+To see only the topic configuration as reported by Redpanda:
+
+[,bash]
+----
+kubectl get topic -n redpanda -o go-template='
+{{- printf "%-30s %-30s %-30s\n" "KEY" "VALUE" "SOURCE" -}}
+{{- range .items -}}
+{{- range .status.topicConfiguration -}}
+{{- printf "%-30s %-30s %-30s\n" .name .value .source -}}
+{{- end -}}
+{{- end -}}
+'
+----
+
+.Example output
+[%collapsible]
+====
+[.no-copy]
+----
+KEY                            VALUE                          SOURCE
+compression.type               producer                       DEFAULT_CONFIG
+cleanup.policy                 compact                        DYNAMIC_TOPIC_CONFIG
+segment.bytes                  67108864                       DEFAULT_CONFIG
+retention.ms                   604800000                      DEFAULT_CONFIG
+retention.bytes                -1                             DEFAULT_CONFIG
+message.timestamp.type         CreateTime                     DEFAULT_CONFIG
+max.message.bytes              1048576                        DEFAULT_CONFIG
+redpanda.remote.read           false                          DEFAULT_CONFIG
+redpanda.remote.write          false                          DEFAULT_CONFIG
+retention.local.target.bytes   -1                             DEFAULT_CONFIG
+retention.local.target.ms      86400000                       DEFAULT_CONFIG
+redpanda.remote.delete         true                           DEFAULT_CONFIG
+segment.ms                     1209600000                     DEFAULT_CONFIG
+----
+====
+
+== Update a topic
+
+To update a topic, edit the Topic resource configuration, and apply the changes.
+
+CAUTION: Do not use rpk or any other Kafka clients to edit topics that you created using the Topic resource. The topic controller monitors the Redpanda cluster for changes to topics. If you use a Kafka client to edit the topic, the topic controller detects those changes as a drift from the desired state and attempts to reconcile by reverting the changes to match the Topic resource's state.
+
+The following example changes the cleanup policy for the `chat-room` topic:
+
+.`example-topic.yaml`
+[,yaml,lines=8-9]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Topic
+metadata:
+  name: chat-room
+spec:
+  partitions: 3
+  replicationFactor: 3
+  additionalConfig:
+    cleanup.policy: "delete"
+  kafkaApiSpec:
+    brokers:
+      - "redpanda-0.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-1.redpanda.<namespace>.svc.cluster.local:9093"
+      - "redpanda-2.redpanda.<namespace>.svc.cluster.local:9093"
+    tls:
+      caCertSecretRef:
+        name: "redpanda-default-cert"
+        key: "ca.crt"
+----
+
+[,bash]
+----
+kubectl apply -f example-topic.yaml --namespace <namespace>
+----
+
+You can also remove a property that overrides the default setting, and the setting will use the default value again.
```

Comment:
read this 3 times and it remains unclear... are you saying that you can replace the default setting this one time and then the prop will revert back to the default next time?

Reply:
Yes 😄 I've clarified that sentence now, thanks.